.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+---
+name: "TER deployment"
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
By the way, would it make sense to use the [release event](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release) instead?